### ISSUE TITLE
415608AF - Goldeneye: 007 Reloaded

### ISSUE THREAD

**Issue by FitterSpace** [xenia-project/game-compatibility#637]
[Marketplace](http://marketplace.xbox.com/en-US/Product/GoldenEye-Reloaded/66acd000-77fe-1000-9115-d802415608af?nosplash=1)

Tested on 9457c63 (Sep 7 2016) from the main page of xenia.jp
https://ci.appveyor.com/project/benvanik/xenia/build/artifacts

# Issues:
The game will run perfectly normal until 25 seconds in, when it will crash without fail. If i mash A through the menus, I can get it to the loading screen for the first level but it will crash before that finishes.
Aside from crashing randomly, the menus and intro run as expected.
A video of Goldeneye: 007 Reloaded running on Xenia can be seen [here](https://www.youtube.com/watch?v=IArYIWxJAKA)

# Log:
[Log file](https://www.dropbox.com/s/d7d10ahon08uqa0/xenia.log?dl=0)

# Labels:
recommended label: gameplay

**Comment by DrChat** [comment]
Looks like it's just hanging after a while.
A few other games hang somewhat similar to this as well, but not as specific as 25 seconds in.

**Comment by Razzile** [comment]
lol I didn't read the description. my bad

**Comment by FitterSpace** [comment]
yeah it also doesn't actually get to any gameplay, either. I am able to get past the loading screen for MI6 Ops in under 25 seconds but it doesn't render anything. Then the game just crashes. But other than that, the menus work as expected. The colors are all crazy but that's about it.

**Comment by Septimus-Prime** [comment]
Where is the file now?

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/ba7dc6b2d7187a233d7175c75994828b498bec58)

Shows only gray screen. Enters menus.

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2173986/xenia.zip)

**Comment by FitterSpace** [comment]
https://youtu.be/bO_iyOtHue8

Tested on Xenia DirectX12 131525e4, October 4, 2018. GoldenEye: 007 Reloaded goes in-game and one of the MI6 ops missions can be loaded. The menus work as expected, although there are some graphical issues with the menu selection (the option you have selected normally glows but on Xenia it doesn't). The MI6 ops mission I loaded crashes once you get to a certain spot in the map, though. I cannot go much farther than 10 feet or so away from the spawn point.

[xenia.log on google drive](https://drive.google.com/file/d/1QWkN77p0k8Y3ao94Or_7KpqmbTUIiaxw/view?usp=sharing)
[xenia.log uploaded to github](https://github.com/xenia-project/game-compatibility/files/2446939/xenia.log)

I apologize for deleting my old log file from the last time I tested this game. I was cleaning out some old files to make room for other things on dropbox, but that shouldn't be an issue in the forseeable future. I won't delete log files anymore.

![image](https://i.gyazo.com/3bd8a3e2d4eb1dbf4f95c2331187debb.jpg)

**Comment by FitterSpace** [comment]
https://youtu.be/hMIBxIwOiJU

[Log File](https://github.com/xenia-project/game-compatibility/files/3645251/Goldeneye.reloaded.September.23.zip)


![image](https://user-images.githubusercontent.com/22065181/65482015-46a42c80-de5d-11e9-8c36-1f887cf75453.png)


This game has had some minor improvements since that last time I tested it. First, the framerate is much better. I’m using the same PC as I was last year, but here I’m able to get 60 frames per second consistently with the gameplay I’ve been able to reach here. 

Unfortunately, the gameplay doesn’t go too far because of frequent crashes, most of which happen before any gameplay is possible. However, there is one single MI6 ops mission I am able to load and play, as long as I don’t look to the left (relative to the direction I’m facing at spawn). For whatever reason, Xenia really hates whatever happens when I look to the left. I’m fairly confident that it’s caused by a loading issue because I can avoid the crash as long as I’m looking down at the ground. In short, the crash doesn’t happen when I look forward or to the right. If I have to turn to the left, I need to look down or else the game will crash. The game is also rather unstable at this point, so it can still crash when things happen. The framerate dropped down a lot when I killed an enemy one time and the game totally locked up when I blew up a truck.

The campaign, the tutorial, and other MI6 missions will crash during the loading screen or during the fade-in right after the loading screen. I cannot test the local splitscreen multiplayer at the moment.

Aside from the crashes, the game looks really good! There are some differences to how it looks on console, but I'm impressed with how accurate the graphics are already. The biggest issues I've noticed here are some shadows are a little off, the bullets don't have tracers, and I saw an enemy with messed up textures one time, but unfortunately I couldn't get that to happen again. Also, there are missing graphics in the menus.

**Comment by Sherm1984** [comment]
Getting a hang at startup

[xenia.log](https://github.com/xenia-project/game-compatibility/files/4532555/xenia.log)

**Comment by FitterSpace** [comment]
This game is almost 100% playable with perfect performance. Unfortunately, there is one glaring issue that makes it impossible to finish the game, at least on my machine. There is a softlock that happens on the last level, which is really disappointing because I was having a blast up until then. But I can't overstate how well the game runs and how minor the issues I faced were. However, rather than talking about the 99.9% of the game that looks and sounds perfect as far as I can tell, I'm going to talk about some of the smaller issues that I'm certain are different to how the game plays on console.
# Issues:
The first issue I noticed was in the third level, Airfield. At the part I'm standing at in this screenshot, I saw some weird green and yellow flickering things. They were solid neon colors that filled up most of the screen. I don't know what they were and I can't get them to appear again, so I don't have a screenshot of that. They also only appeared for a few frames then they would go away for a second or two. Sort of like a strobe light, except slightly less annoying.

![Screenshot 2020-10-12 232950](https://user-images.githubusercontent.com/22065181/95815689-d758df00-0ce2-11eb-9436-4e8dae149c77.png)

The next issue was a disappearing floor in the Outpost level. This was the biggest single issue I ran into besides the softlock at the end of the game. Like 007 Legends, some bits of terrain randomly glitch out or disappear entirely. Although it's much less common in this game, it's very noticeable in this level. It would be hard to finish this level casually since there are some invisible objects you have to maneuver around, almost like a maze.
![Untitled3](https://user-images.githubusercontent.com/22065181/95816222-2a7f6180-0ce4-11eb-89c9-d0a9f643d4e2.png)
![Untitled2](https://user-images.githubusercontent.com/22065181/95815541-86e18180-0ce2-11eb-913d-cf2b76d75d6b.png)



Going back a few levels, the same problem exists in Nightclub, but it's not as drastic. All the objects that disappear are still there, so you can't walk through them or anything. But it can be hard to navigate some parts if you don't already know where to go. There's also a weird dithering pattern on the girl's body that shouldn't be there. Not sure what that's all about. This is the only time I noticed any glitches that look like that.
![Untitled6](https://user-images.githubusercontent.com/22065181/95816004-9f9e6700-0ce3-11eb-8a6e-ed453f1e77e3.png)
![Untitled5](https://user-images.githubusercontent.com/22065181/95816008-a036fd80-0ce3-11eb-983f-dda944a4b008.png)

Here's a graphical issue that happens everywhere in the game, but this is the craziest example of it. This is not an explosion. There is a bright light source on that cargo lift, and Xenia is exaggerating the brightness of it. Speaking of, the emulator doesn't accurately handle the lighting in this game. Every part of the game is noticeable brighter than it should be. Even the parts where you're supposed to use night vision to see anything can be easily navigated without them.
![Untitled4](https://user-images.githubusercontent.com/22065181/95816400-a1b4f580-0ce4-11eb-8855-db5892a1a803.png)



This is the only real gameplay difference I noticed while playing the game. In this part of the Archives mission, you are supposed to stand your ground while you survive a few waves of enemies. However, the enemies get stuck in their spawn point and never come out.
![Untitled](https://user-images.githubusercontent.com/22065181/95816249-3cf99b00-0ce4-11eb-8c28-f6772b3cbcbf.png)

Some other minor issues I've noticed is that sometimes you'll get stuck in a door as it's opening. Not sure what could be causing that to happen. [video example](https://youtu.be/9zpjbPy6Sx4?t=140)

Another minor bug is that looking all the way down and crouching will cause the camera to twitch a little bit.

Last, but not least, there is a crippling issue with the final level of the game. In this level, you're supposed to protect Natalya while she disables the GoldenEye satellite. However, she doesn't stop at the last console and instead runs straight past it and into the corner, where she gets stuck forever. I haven't found a way to stop this from happening. It's a shame, because I'd consider this game 99% playable if it weren't for that.
![Screenshot 2020-10-12 230027](https://user-images.githubusercontent.com/22065181/95816588-1720c600-0ce5-11eb-820f-9eaa554da9bb.png)

Overall, this game is extremely playable and I'm really surprised at how well it performs on my mid-range PC. It's actually pretty insane how much this game has improved in Xenia over the past year. 

## Video:
https://youtu.be/9zpjbPy6Sx4

## Log File (93.1 MB zipped, 4.9 GB uncompressed):
https://www.dropbox.com/s/c16a2p7djp1e58z/GoldenEye_Log.zip?dl=0
This log file is for my full playthrough and is unrelated to the video above.

**Comment by notimp** [comment]
FYI - vsync needs to be enabled, otherwise firing a gun (right trigger) only registers one out of 20 times. :)

**Comment by CougarYap** [comment]
The game was running smooth around 60 fps, huge lag spikes when loading in game. But crash a lot when I pressed any button, it won't let me play for long.

[xenia.log](https://github.com/xenia-project/game-compatibility/files/6248054/xenia.log)

**Comment by FitterSpace** [comment]
Tested on https://github.com/xenia-project/xenia/commit/360e2f541

Everything I said in my last post still applies here, but the game is much less stable in this version of Xenia. I've attached a video showing what I mean and a few log files that end with the guest crashing.

## Video:

https://youtu.be/lHnLoTBl2Q0

## Log Files:

[GEReloadedCrash.zip](https://github.com/xenia-project/game-compatibility/files/7214796/GEReloadedCrash.zip) (388 MB uncompressed)
[GEReloadedCrash2.zip](https://github.com/xenia-project/game-compatibility/files/7214797/GEReloadedCrash2.zip) (382 MB uncompressed)
[GEReloadedCrash3.zip](https://github.com/xenia-project/game-compatibility/files/7214798/GEReloadedCrash3.zip) (210 MB uncompressed)

**Comment by adit1110** [comment]
Hi there,

I have been having this issue where the whole game freezes and it gives me this error message at around 10-15 minutes of playtime per session. I have gone onto the Xenia discord server's help channel and someone there told me to update the emulator by installing a new version, but it did not help. 

### My system:

- Xenia - master version
- GTX 1650
- i5-9300H
- 8GB RAM

### Error in question
![Screenshot_243](https://user-images.githubusercontent.com/84141926/152623497-d1dcb1a6-4393-43e7-8be7-6524f8ba2352.png) 

### Log File
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/8007094/xenia.zip) 

Any help would be appreciated.

**Comment by TapeWerm** [comment]
Lag when I use V sync seems to be gone on Nvidia driver 512.59 on RTX 2060. I can't remember when said lag started.

**Comment by HALT77** [comment]
Still have a lot of lightining problems, and the button to shoot does not seem to register every time... and clicking in the MI6 ops in the menu will make the game crash

**Comment by UnAssumedAlias** [comment]
Game still crashes here and there randomly, but otherwise it runs perfectly with vsync off.

**Comment by tvoracek** [comment]
Always crashes for me at the Nightclub when the guy starts talking and gets closer to doors. Bam crash.

**Comment by austinarchibald** [comment]
Trying to play local multiplayer. Keep getting guest crash error within a couple minutes of any game. Sometimes happens even in the menu set up while picking weapons. Unfortunate.

**Comment by gir489returns** [comment]
This game is completely unplayable. Crashes after about 3 minutes.

**Comment by austinarchibald** [comment]
Yes, use RPCS3 for Goldeneye, Xenia for RDR1. Been running great with minimal issues.

**Comment by IIITTIII** [comment]
> This game is completely unplayable. Crashes after about 3 minutes.

it appears this happened when stuff in game don't load fast enough. it does crash as said even then on latest canary, possibly quick save or autosave related to crashing. reverting to checkpoint as well crash's. been playing and doing speed run into areas and in definitely its loading is causing crashing.

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/63
